stable/jenkins/values.yaml

```diff
@@ -14,6 +14,14 @@
 clusterZone: "cluster.local"
 
 master:
+  httpsKeyStore:
```

Comment:
Would be great if we could also support existing secrets.

Reply:
Would you please elaborate on what do you mean exactly by existing secrets?

Reply:
I mean instead of templating the Secret {{ template "jenkins.fullname" . }}-https-jks to have the option to just provide the name of an existing secret. As long as that contains the required fields it should work.

This avoids having to pass credentials as part of helm values.

Reply:
> I mean instead of templating the Secret {{ template "jenkins.fullname" . }}-https-jks to have the option to just provide the name of an existing secret. As long as that contains the required fields it should work.
> 
> This avoids having to pass credentials as part of helm values.

Done, please check if that's what you ask